plugins/greatexpectations/setup.py

```diff
@@ -0,0 +1,34 @@
+from setuptools import setup
+
+PLUGIN_NAME = "greatexpectations"
```

Comment:
Could we update this to `great_expectations` as well?

Reply:
Done.